controllers/controllers.js

```diff
@@ -0,0 +1,15 @@
+const { selectTopics } = require("../models/models.js");
+
+exports.pathInvalid = (req,res) =>{
+  res.status(404).send({message: "Invalid API path request"})
+}
+
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch((err) => {
+      next(err);
```

Comment:
if you like you can clean this up a tad by just writing:
`.catch(next);` 
express will read this the same way and invoke `next` with the `err` for you